src/puppet.js

```diff
@@ -653,6 +780,11 @@
     }
 
     this.queue.receive(this.obj, patches);
+    if(this.queue.pending && this.queue.pending.length && this.queue.pending.length > 1) {
```

Comment:
Agree about hard-coding. Good point about keystrokes, will test it with some textboxes